rero/rero-ils-ui
# document: display url in detail view

* Adds url fields according to specifications.
* Corrects edition statement display.
* Adds class 'label-title' and replaces translate pipe by directive.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

Implements task 1319 of US1262
https://tree.taiga.io/project/rero21-reroils/task/1319?kanban-status=1224894

## How to test?

Needs https://github.com/rero/rero-ils/pull/761
Go to document detailed view and check display
Usefull PIDs: 
* displayed below 'subjects': 37, 57, 67, 104
* displayed like a holding: 71, 150, 165, 227

## Code review check list
